src/cmd/manage_nsfs.js

```diff
@@ -758,6 +754,13 @@ async function validate_bucket_args(data, action) {
         if (!exists) {
             throw_cli_error(ManageCLIError.InvalidStoragePath, data.path);
         }
+        const account = await verify_bucket_owner(data.bucket_owner, action);
+        if (action === ACTIONS.ADD) data.owner_account = account._id;
```

Comment:
A variable assignment doesn't fit a validation function, why did you move it here?

Reply:
I remembered that we wanted fewer calls to the validations function inside the operation, in this case, I tried to reduce the call to `verify_bucket_owner` from `add_bucket`.
In the previous, I returned the account id, and now I changed to the account since I need the `account.nsfs_account_config` and thought that by that I can reduce a call to read the account config file again.

Reply:
still, I don't think this is the place for assigning a property of the account

Reply:
I removed the assignment, thanks.